algebra/src/main/scala/endpoints/algebra/Responses.scala

```diff
@@ -0,0 +1,20 @@
+package endpoints.algebra
+
+import scala.language.higherKinds
+
+trait Responses {
+
+  /** Information carried by a response */
+  type Response[A]
+
+  /**
+    * Empty response.
+    */
+  def emptyResponse: Response[Unit]
+
+  /**
+    * String response.
+    */
+  def stringResponse: Response[String]
```

Comment:
I would name it `textResponse` to emphasize the fact that the generated response will have content type `plain/text`.